src/main/java/it/gov/pagopa/payhub/activities/mapper/ingestionflow/debtposition/InstallmentSynchronizeMapper.java

```diff
@@ -0,0 +1,107 @@
+import it.gov.pagopa.pu.debtposition.dto.generated.InstallmentSynchronizeDTO;
+import it.gov.pagopa.pu.debtposition.dto.generated.TransferSynchronizeDTO;
+import org.apache.commons.collections4.MultiValuedMap;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.IntStream;
+
+@Service
+@Lazy
+public class InstallmentSynchronizeMapper {
+
+    public InstallmentSynchronizeDTO map(InstallmentIngestionFlowFileDTO installmentIngestionFlowFileDTO,
+                                         Long ingestionFlowFileId,
+                                         Long organizationId) {
+        return InstallmentSynchronizeDTO.builder()
+                .ingestionFlowFileId(ingestionFlowFileId)
+                .ingestionFlowFileLineNumber(installmentIngestionFlowFileDTO.getIngestionFlowFileLineNumber())
+                .organizationId(organizationId)
+                .action(InstallmentSynchronizeDTO.ActionEnum.valueOf(installmentIngestionFlowFileDTO.getAction().name()))
+                .draft(installmentIngestionFlowFileDTO.getDraft())
+                .iupdOrg(installmentIngestionFlowFileDTO.getIupdOrg())
+                .description(installmentIngestionFlowFileDTO.getDescription())
+                .validityDate(installmentIngestionFlowFileDTO.getValidityDate())
+                .multiDebtor(installmentIngestionFlowFileDTO.getMultiDebtor())
+                .notificationDate(installmentIngestionFlowFileDTO.getNotificationDate())
+                .paymentOptionIndex(installmentIngestionFlowFileDTO.getPaymentOptionIndex())
+                .paymentOptionType(installmentIngestionFlowFileDTO.getPaymentOptionType())
+                .paymentOptionDescription(installmentIngestionFlowFileDTO.getPaymentOptionDescription())
+                .iud(installmentIngestionFlowFileDTO.getIud())
+                .iuv(installmentIngestionFlowFileDTO.getIuv())
+                .entityType(InstallmentSynchronizeDTO.EntityTypeEnum.valueOf(installmentIngestionFlowFileDTO.getEntityType().name()))
+                .fiscalCode(installmentIngestionFlowFileDTO.getFiscalCode())
+                .fullName(installmentIngestionFlowFileDTO.getFullName())
+                .address(installmentIngestionFlowFileDTO.getAddress())
+                .civic(installmentIngestionFlowFileDTO.getCivic())
+                .postalCode(installmentIngestionFlowFileDTO.getPostalCode())
+                .location(installmentIngestionFlowFileDTO.getLocation())
+                .province(installmentIngestionFlowFileDTO.getProvince())
+                .nation(installmentIngestionFlowFileDTO.getNation())
+                .email(installmentIngestionFlowFileDTO.getEmail())
+                .dueDate(installmentIngestionFlowFileDTO.getDueDate())
+                .amount(installmentIngestionFlowFileDTO.getAmount())
```

Comment:
what about to convert amount into cents and send to debt-position API already as Cents?

@Benedetta-fabbri what do you think about it?